test/common/quic/envoy_quic_h3_fuzz_helper.h

```diff
@@ -0,0 +1,85 @@
+#include <set>
+
+#include "test/common/quic/envoy_quic_h3_fuzz.pb.h"
+
+#include "quiche/quic/core/crypto/null_decrypter.h"
+#include "quiche/quic/core/crypto/null_encrypter.h"
+#include "quiche/quic/core/quic_connection.h"
+#include "quiche/quic/core/quic_versions.h"
+
+namespace Envoy {
+namespace Quic {
+
+// This class serializes structured protobuf `HTTP/3` messages to bytes as they
+// would be sent over the wire.
+class H3Packetizer {
+public:
+  H3Packetizer(std::set<uint32_t>& streams) : open_h3_streams_(streams){};
+  size_t serialize(char* buffer, size_t buffer_len, bool unidirectional, uint32_t type, uint32_t id,
+                   const test::common::quic::H3Frame& h3frame);
+
+private:
+  std::set<uint32_t>& open_h3_streams_;
+};
+
+// This class serializes structured protobuf `QUIC + HTTP/3` messages to bytes as they
+// would be sent over the wire.
+class QuicPacketizer {
+public:
+  QuicPacketizer(const quic::ParsedQuicVersion& quic_version,
+                 quic::QuicConnectionHelperInterface* connection_helper);
+  void serializePackets(const test::common::quic::QuicH3FuzzCase& input);
+  void foreach (std::function<void(const char*, size_t)> cb);
+  void reset();
+
+private:
+  void serializePacket(const test::common::quic::QuicFrame& frame);
+  void serializeJunkPacket(const std::string& data);
+
+  quic::ParsedQuicVersion quic_version_;
+  quic::QuicConnectionHelperInterface* connection_helper_;
+  quic::QuicPacketNumber packet_number_;
+
+  quic::QuicConnectionId destination_connection_id_;
+  quic::QuicFramer framer_;
+
+  H3Packetizer h3packetizer_;
+  std::set<uint32_t> open_h3_streams_;
```

Comment:
Can you comment about how this field is used? It's not very clear to me by reading the code.